test/unit/grpc-fallback.ts

```diff
@@ -129,7 +132,8 @@ describe('grpc-fallback', () => {
     echoService: protobuf.Service,
     stubOptions: {};
   const createdAbortControllers: string[] = [];
-  const savedAbortController = abortController.AbortController;
+  // @ts-ignore
```

Comment:
Please do not use `@ts-ignore` :) We may need to do some casting, but a full scale ignore is almost never the safest thing to do.  It's already used throughout these tests, so I won't block on it.